logstreams/src/main/java/io/zeebe/logstreams/log/LogStreamBuilder.java

```diff
@@ -20,7 +20,7 @@
 
 @SuppressWarnings("unchecked")
 public class LogStreamBuilder<SELF extends LogStreamBuilder<SELF>> {
-  private static final int MINIMUM_FRAGMENT_SIZE = 32 * 1024;
+  private static final int MINIMUM_FRAGMENT_SIZE = 4 * 1024;
```

Comment:
:man_shrugging: :sweat_smile:

Reply:
We can remove a minimum constraint I guess :man_shrugging: but then people may put 1 byte or something :man_shrugging:

Reply:
was wondering why you decreased the default value ^^

Reply:
Ah, I ran into some tests where it was set to lower, but now I can't remember.